src/test/java/org/example/DeliveryToFunctionality.java

```diff
@@ -21,7 +22,8 @@ public class DeliveryToFunctionality {
 
```

Comment:
Before @Before Method you should set up for example: "private WebDriver webdriver" - without it this code doesn't work, because it doesn't recognize "webdriver".